znnl/training_strategies/training_decorator.py

```diff
@@ -92,6 +92,17 @@ def wrapper(trainer: "SimpleTraining", *args, **kwargs):
         Output of the train_fn. A more detailed documentation can be found in the used
         training strategy.
         """
+        # Check whether a model is available
+        if trainer.model is None:
+            raise ValueError("No model available. Please initialize a model first.")
+        # Check whether the model has been set up properly
+        if trainer._train_step is None:
+            raise ValueError(
+                "The model has not been set up properly."
```

Comment:
This exception isn't very helpful from my perspective. If I were reading this after a model failed I wouldn't really know what to call to fix it. Can you add more information like` call model.set_model()` or something more direct.

Reply:
Fixed in [6f75030](https://github.com/zincware/ZnNL/pull/105/commits/6f75030536a93d9c5a2279f433b9fb00e756d5e2)